lib/cusolver/cusolverRF.jl

```diff
@@ -0,0 +1,429 @@
+
+import CUDA.CUBLAS: unsafe_batch, unsafe_strided_batch
+
+function cusolverRfCreate()
+    handle_ref = Ref{cusolverRfHandle_t}()
+    cusolverRfCreate(handle_ref)
+    return handle_ref[]
+end
+
+function cusolverRfFree(handle)
+    if handle != C_NULL
+        cusolverRfDestroy(handle)
+        handle = C_NULL
+    end
+end
+
+mutable struct RfHandle
+    handle::Ptr{cusolverRfHandle_t}
+end
+
+function RfHandle(;
```

Comment:
We typically make handles task-local, and cache them. See e.g. `dense_handle()` in `CUSOLVER.jl`. Maybe `sparse_rf_handle` would be a good name for this function, to fit in with the others.